src/components/Common/Table/Table.js

```diff
@@ -184,10 +189,10 @@ export const Table = observer(
                 modal({
                   title: "Source for task " + out?.id,
                   style: { width: 800 },
-                  body: <TaskSourceView content={out} onTaskLoad={onTaskLoad} />,
+                  body: <TaskSourceView content={out} onTaskLoad={onTaskLoad} sdkType={type} />,
                 });
               }}
-              icon={<Icon icon={FaCode}/>}
+              icon={isFF(FF_LOPS_E_3) ? <Icon icon={RiCodeSLine} style={{ width: 24, height: 24 }}/> : <Icon icon={FaCode}/>}
```

Comment:
I think the icon that matches the designs here would be https://react-icons.github.io/react-icons/icons?name=ri#RiCodeLine not the spaced version.

Reply:
good catch - i'll update right away